src/styles/productPage.css

```diff
@@ -0,0 +1,93 @@
+
+
+.ProductPage-container{
```

Comment:
tên class không nên đặt ký tự in hoa
phải có dấu cách giữa tên class và scope. Ví dụ: .aaa {...}